homeassistant/components/streamlabswater/__init__.py

```diff
@@ -0,0 +1,112 @@
+"""
+Support for Streamlabs Water Monitor.
+
+This component provides sensors that return the daily, monthly, and yearly
+water usage as returned by the Streamlabs Water service. A binary sensor
+is used to indicate whether the water monitor is in the home or away mode.
+
+In addition to the sensors, a service is provided that can be used to set
+the away mode of the water monitor.
+
+The minimum configuration needed to get started is:
+
+    streamlabswater:
+      api_key: <your_api_key>
+
+where the api_key is retrieved by following the instructions at:
+https://developer.streamlabswater.com/docs/getting-started.html
+
+By default the first location found will be used. You can specify a location:
+
+    streamlabswater:
+      api_key: <your_api_key>
+      location_id: <your_location_id>
+
+The away mode service is exposed at streamlabswater.set_away_mode where the
+required parameter away_mode should be set to either home or away.
+"""
```

Comment:
The content of the docstring should be move to the documentation.

Reply:
Done. Made this consistent with other integrations now that documentation is open for this change.